### PR TITLE
[DT-191][risk=no] Rebuild Hotfix CDR v5 - reverting code changes since 5/2022

### PR DESCRIPTION
**Pull request by freemabd**
Rebuild Hotfix CDR v5 - reverting code changes since 5/2022

### CODE REVIEW

File: api/db-cdr/generate-cdr/create-tables.sh [hunk 1]
```diff
@@ -1,77 +1,69 @@
...
 schema_path=generate-cdr/bq-schemas
 for filename in generate-cdr/bq-schemas/*.json;
 do
     json_name=${filename##*/}
     table_name=${json_name%.json}
+    if [[ "$table_name" != 'ds_procedure_occurrence_52' ]]
+    then
+      echo "Deleting $table_name"
+      bq --project_id="$BQ_PROJECT" rm -f "$BQ_DATASET.$table_name"
+    fi
```
Comment (chenchals) [inline api/db-cdr/generate-cdr/create-tables.sh:29]: Need to double check if bq-schemas contain the ds_procedure_occurrence_52.json, as it is not present in main

**Review by chenchals — APPROVED** [review]
Please check for bq-schemas/ds_procedure_occurrence_52.json, as it is not present in main and we reference it in create-tables.sh.

**Comment by chenchals** [comment]
also please check for the presence of 'QUALIFIER_SOURCE_VALUE` for ds_procedure_occurrence_52.json

File: api/db-cdr/generate-cdr/create-tables.sh (hunk 1 above)
Comment (freemabd) [reply api/db-cdr/generate-cdr/create-tables.sh:29]: Thanks for noticing this. Added.

**Review by chenchals — APPROVED** [review]
LGTM!

**Pull request merged**